### PR TITLE
search-bar

### PR DESCRIPTION
**Pull request by SergeyShapovalenko**
[DEMO LINK](https://SergeyShapovalenko.github.io/layout_search-bar-airbnb/)
[TEST REPORT LINK](https://SergeyShapovalenko.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+.input--small {
+  padding-left: 33px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  margin-top: 20px;
+  left: 48px;
+  top: 481px;
```
Comment (tetiana-ivanchenko) [inline src/style.css:50]: is it works values without positioning?

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  border-radius: 4px;
+}
+
+.input--big {
+  font-size: 16px;
+  line-height: 22px;
+  margin-top: 20px;
+  height: 70px;
+  left: 48px;
+  top: 71px;
```
Comment (tetiana-ivanchenko) [inline src/style.css:37]: is it works values without positioning?

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  left: 62px;
+  top: 25px;
+}
+
+.input--small::placeholder {
+  font-weight: 400;
+  font-size: 14px;
+  color: #3d4e61;
+  left: 33px;
+  top: 12;
```
Comment (tetiana-ivanchenko) [inline src/style.css:68]: is it works values without positioning?

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+  left: 48px;
+  top: 481px;
+  border-radius: 4px;
+  height: 42px;
+}
+
+.input--big::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
```
Comment (tetiana-ivanchenko) [inline src/style.css:58]: you should move repeatable color in thw variables

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+  left: 48px;
+  top: 71px;
+  border-radius: 4px;
+  padding-left: 62px;
+}
+
+.input--small {
+  padding-left: 33px;
+  font-style: normal;
+  font-weight: 400;
```
Comment (tetiana-ivanchenko) [inline src/style.css:45]: it is by default

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+form {
+  position: relative;
+}
+
+.input {
+  font-family: Avenir, Arial, sans-serif;
+  display: block;
+  width: 100%;
+  font-weight: 300;
+  background: #fff;
```
Comment (tetiana-ivanchenko) [inline src/style.css:25]: it is by default

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+.input--big::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  left: 62px;
+  top: 25px;
+}
+
+.input--small::placeholder {
+  font-weight: 400;
```
Comment (tetiana-ivanchenko) [inline src/style.css:64]: it's default value

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,89 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
```
Comment (agniya-i) [inline src/style.css:7]: add `font-style` and `font-weight` (checklist p7)

**Review by agniya-i — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+  line-height: 22px;
+  margin-top: 20px;
+  height: 70px;
+  border-radius: 4px;
+  padding-left: 62px;
+}
+
+.input--small {
+  padding-left: 33px;
+  font-style: normal;
```
Comment (DarkMistyRoom) [inline src/style.css:36]: `font-style: normal;` is the default value. You can remove it

File: src/style.css [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+  top: 25px;
+}
+
+.second-searcher {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+}
```
Comment (DarkMistyRoom) [inline src/style.css:89]: Don't repeat the styles. Check the 2nd point in the checklist

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+  font-size: 16px;
+  line-height: 22px;
+  color: var(--placeholder-color);
+}
+
+.input--small::placeholder {
+  font-weight: 400;
+  font-size: 14px;
+  color: var(--placeholder-color);
+}
```
Comment (DarkMistyRoom) [inline src/style.css:57]: don't use repeated styles. you can set them to `.input:placeholder`

Comment (DarkMistyRoom) [reply src/style.css:57]: check repeating everywhere

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 12]
```diff
@@ -0,0 +1,90 @@
...
+  font-weight: 300;
+  font-style: normal;
+}
+
+form {
+  position: relative;
+}
+
+.input {
+  font-family: Avenir, Arial, sans-serif;
```
Comment (SanyaBratashchuk) [inline src/style.css:17]:
```suggestion
  font-family: inherit;
```
Move font-family to the `html` tag selector

**Review by SanyaBratashchuk — APPROVED** [review]

**Pull request closed without merging**